test/data/tsang-falloff.yaml

```diff
@@ -0,0 +1,224 @@
+generator: cti2yaml
+cantera-version: 2.6.0a2
+date: Mon, 30 Aug 2021 16:33:51 +0300
+input-files: [Lucassen-2011.cti]
+
+units: {length: cm, quantity: mol, activation-energy: cal/mol}
+
+phases:
+- name: gas
+  thermo: ideal-gas
+  elements: [H, C, O, N, Ar]
+  species: [H, NO, OH, CN, HCN, HONO, CO2, H2O, N2, N2O]
```

Comment:
You could shorten this input file significantly by borrowing 'species' from another file:
```yaml
  species:
  - gri30.yaml/species: [H, NO, OH, CN, HCN, CO2, H2O, N2, N2O]
  - species: [HONO]
```
which means that you could eliminate all species entries except for `HONO` below

Reply:
I'm not getting the syntax exactly right - `gri30.yaml` is in `../data/` relative to this mechanism and that path seems to be causing an error.
Would it be acceptable to leave this as is to have a self-contained test mechanism?

Reply:
gri30 is always available on an internal search path, so no path should be required here. The code snippet given above should work? (Replacing what is currently line 14)

Reply:
I made a mistake before - thanks for making the change explicit. It works now